integration_test/README.md

```diff
@@ -24,9 +24,12 @@ the test will:
 1.  Configure the application to expose its metrics by running
     `applications/<application>/<platform>/post` on the VM. This might
     be a no-op for some applications.
-1.  Configure the the Ops Agent to look for the application's metrics by
-    running
-    `agent/ops-agent/<platform>/enable_<application>` on the VM.
+1.  Configure the the Ops Agent to look for the application's logs/metrics by
+    running `agent/ops-agent/<platform>/enable_<application>` on the VM.
+1.  run `exercise` script
```

Comment:
What's the difference between line 29 and line 71 - 72?

Reply:
1. I forgot to expand "run exercise script" into a real sentence; this is done now.
2. the two mentions of the  "exercise" script are in different sections. Expand the diff to see that there is a lot of stuff in between and this isn't just one big list. :)

Reply:
Ah I see. Yeah, I got confused by way the GitHub displays it. Now I get it. Thanks for the clarification!